Cubical/Categories/Instances/Preorders/Monotone/Adjoint.agda

```diff
@@ -41,27 +40,29 @@ module _ {ℓ ℓ' : Level} where
   HasLeftAdj : {X Y : Preorder ℓ ℓ'} → (f : MonFun X Y) → Type ((ℓ-max ℓ ℓ'))
   HasLeftAdj {X} {Y} f = Σ[ L ∈ MonFun Y X ] (L ⊣ f)
 
-  open IsPoset
+  open isUnivalent
+  open IsPreorder
   open _⊣_
   open Iso
 
   -- when assuming that the relation on X is antisymmetric,
   -- we get that adjoints are unique
   isPropHasLeftAdj : {X Y : Preorder ℓ ℓ'}
```

Comment:
Hopefully we can eventually derive this from a more general proof about functors to reduce the duplication.

Reply:
Agreed. Would be good to chat about how to grab the categorical concepts and "thin" them in general.